### PR TITLE
edit arabic localization

### PR DESCRIPTION
**Pull request by 3b7ameed**
@erciccione

### CODE REVIEW

File: translations/monero-core_ar.ts [hunk 1]
```diff
@@ -74,28 +74,28 @@
...
+        <translation>الأمر + نزر انتر (مثال help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="93"/>
         <source>Starting local node in %1 seconds</source>
-        <translation type="unfinished"></translation>
+        <translation>بدأ تشغيل الخادم المحلي في خلال % ثواني</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: Please add the place marker (`%1`) correctly

File: translations/monero-core_ar.ts [hunk 2]
```diff
@@ -283,7 +283,7 @@
     <message>
         <location filename="../components/HistoryTableMobile.qml" line="50"/>
         <source>Payment ID:</source>
-        <translation type="unfinished">هويه المعامله</translation>
+        <translation>هويه المعامله</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `:`

File: translations/monero-core_ar.ts [hunk 3]
```diff
@@ -1852,80 +1860,87 @@ The old wallet cache file will be renamed and can be restored later.
...
+        <translation>
+المستلم:</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="568"/>
         <source>
 payment ID: </source>
-        <translation>-هويه المعامله:</translation>
+        <translation>
+هويه المعامله:</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `:`

File: translations/monero-core_ar.ts [hunk 4]
```diff
@@ -298,110 +298,112 @@
...
         <location filename="../components/HistoryTableMobile.qml" line="53"/>
         <source>Destinations:</source>
-        <translation type="unfinished"></translation>
+        <translation>المستلمين:</translation>
     </message>
     <message>
         <location filename="../components/HistoryTableMobile.qml" line="54"/>
         <source>Rings:</source>
-        <translation type="unfinished"></translation>
+        <translation>الحلقات</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `:`

File: translations/monero-core_ar.ts [hunk 5]
```diff
@@ -1037,37 +1039,37 @@
...
         <location filename="../pages/settings/SettingsInfo.qml" line="165"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;a href=&apos;#&apos;&gt; (اضغط للتغيير)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="167"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>تعيين طول إسترجاع جديد</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `:`

File: translations/monero-core_ar.ts [hunk 6]
```diff
@@ -1852,80 +1860,87 @@ The old wallet cache file will be renamed and can be restored later.
...
+        <translation>
+عدد المعاملات:</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="566"/>
         <source>
 Transaction #%1</source>
-        <translation>-معامله #1</translation>
+        <translation>
+معامله #1</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: Fix place marker (`#%1`)

File: translations/monero-core_ar.ts [hunk 7]
```diff
@@ -1852,80 +1860,87 @@ The old wallet cache file will be renamed and can be restored later.
...
 Ringsize: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+حجم الطوق:</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="585"/>
         <source>Confirmation</source>
-        <translation>التاكيدات:</translation>
+        <translation>توكيدات:</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: you can remove the `:` here

File: translations/monero-core_ar.ts [hunk 8]
```diff
@@ -2091,7 +2108,7 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
     <message>
         <location filename="../wizard/WizardConfigure.qml" line="153"/>
         <source>Allow background mining?</source>
-        <translation>إسمح بالتعدين في الخلفيه</translation>
+        <translation>إسمح بالتعدين بالخلفيه</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `?`

File: translations/monero-core_ar.ts [hunk 9]
```diff
@@ -2176,12 +2193,12 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
         <location filename="../wizard/WizardDonation.qml" line="162"/>
         <source>For every transaction, a small transaction fee is charged. This option lets you add an additional amount, as a percentage of that fee, to your transaction to support Monero development. For instance, a 50% autodonation take a transaction fee of 0.005 XMR and add a 0.0025 XMR to support Monero development.</source>
-        <translation>لكل معامله هناك رسوم بسيطه. هذا الخيار يسمح ك باضافه كميه ا	ضافيه , كنسبه من هذه الرسوم , من معاملتك لدعم تطوير مونيرو. علي المثال , 50% تبرع تلقائي من معامله رسومها 0.005 XMR ويضاف 0.0025 XMR لمساعده تطوير مونيرو. </translation>
+        <translation>لكل معامله هناك رسوم بسيطه. هذا الخيار يسمح ك باضافه كميه اضافيه كنسبه من هذه الرسوم من معاملتك لدعم تطوير مونيرو. علي المثال 50% تبرع تلقائي من معامله رسومها 0.005  يضاف 0.0025  لمساعده تطوير مونيرو. </translation>
     </message>
     <message>
         <location filename="../wizard/WizardDonation.qml" line="174"/>
         <source>Allow background mining?</source>
-        <translation>السماح بالتعدين في الخلفيه</translation>
+        <translation>السماح بالتعدين بالخلفيه</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: Missing question mark

**Comment by 3b7ameed** [comment]
@erciccione  all done , yeah i didn't see the ringdb/blackballing part before so i felt like i needed to understand it first so i can translate it

**Comment by erciccione** [comment]
@3b7ameed There are some links who can help you to understand what blackballing means and how it works:
https://github.com/monero-ecosystem/monero-GUI-guide/blob/master/monero-GUI-guide.md#53-shared-ringdb
https://monero-blackball.github.io/monero-blackball-site/
https://monero.stackexchange.com/questions/8225/how-can-i-use-monero-blockchain-blackball-to-improve-my-privacy

Let me know if something is not clear

File: translations/monero-core_ar.ts [hunk 10]
```diff
@@ -56,46 +56,46 @@
...
         <location filename="../components/DaemonManagerDialog.qml" line="93"/>
         <source>Starting local node in %1 seconds</source>
-        <translation type="unfinished"></translation>
+        <translation>بدأ تشغيل الخادم المحلي في خلال %1 ثواني</translation>
     </message>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="111"/>
         <source>Start daemon (%1)</source>
-        <translation>إبدأ الخادم</translation>
+        <translation> إبدأ الخادم (1%)</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: The place marker should be inverted: `(1%)`

File: translations/monero-core_ar.ts [hunk 11]
```diff
@@ -601,27 +603,27 @@
...
-        <translation>التعدين بجهازك بيساعد في تقويه شبكه مونيرو. كل ما ناس ازيد بتعدن  كل ما كان اصعب مهاجمه الشبكه,  وكل مساهمه صغيره بتفرق ,. التعدين الفردي بيديك فرصه تكسب شويه مونيرو ,جهازك هيحاول يحل معادلات البلوك . لو لقيت بلوك والجايزه راحت ليك , مبروك عليك يا ابن المحظوظه </translation>
+        <translation>التعدين بجهازك بيساعد في تقويه شبكه مونيرو. كل ما ناس ازيد بتعدن  كل ما كان اصعب مهاجمه الشبكه,  وكل مساهمه صغيره بتفرق ,. التعدين الفردي بيديك فرصه تكسب شويه مونيرو ,جهازك هيحاول يحل معادلات الكتل . لو لقيت بلوك والجايزه راحت ليك , مبروك عليك يا محظوظ </translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="95"/>
         <source>CPU threads</source>
-        <translation type="unfinished"></translation>
+        <translation>عدد خيوط المعالج:</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: You can remove the `:` here

File: translations/monero-core_ar.ts [hunk 12]
```diff
@@ -697,40 +699,40 @@
...
+        <translation>معلومات</translation>
     </message>
 </context>
 <context>
     <name>NetworkStatusItem</name>
     <message>
         <location filename="../components/NetworkStatusItem.qml" line="43"/>
         <source>Synchronizing</source>
-        <translation>مزامنة:</translation>
+        <translation>جاري المزامنه:</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: Yo can remove the `:`

File: translations/monero-core_ar.ts [hunk 13]
```diff
@@ -1079,391 +1081,403 @@ The following information will be deleted
...
         <location filename="../pages/SharedRingDB.qml" line="189"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>تحميل</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="205"/>
         <source>Or manually blackball/unblackball a single output:</source>
-        <translation type="unfinished"></translation>
+        <translation>او اضافه مخرج مرفوض يدوياً</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: missing `:`

File: translations/monero-core_ar.ts (hunk 10 above)
Comment (3b7ameed) [reply translations/monero-core_ar.ts]: it's alright it looks like that cuz of the arabic words

Comment (3b7ameed) [reply translations/monero-core_ar.ts]: I edited everything else

**Pull request merged**